HW5_numpy/numpy_challenge.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+
+# Create 3 arrays using different methods
+a = np.array([1, 2, 3, 4, 5])
+b = np.zeros(5)
+c = np.arange(13)
+
+# Function matrix_multiplication accepts 2 matrixes and multiplies them
+
+
+def matrix_multiplication(a, b):
+    print(a @ b)
+
+
+# Function multiplication_check accepts a list of matrixes
+# and prints "True", if they could be multiplied as they are ordered in the list,
+# and "False", if they could not be multiplied
+
+
+def multiplication_check(list_mat):
+    for i in range(len(list_mat)-1):
+        if (list_mat[i+1]).shape[0] != (list_mat[i]).shape[1]:
+            return False
+    return True
```

Comment:
Идеально!

Reply:
Только зачем скобочки?
```suggestion
def multiplication_check(list_mat):
    for i in range(len(list_mat)-1):
        if list_mat[i+1].shape[0] != list_mat[i].shape[1]:
            return False
    return True
```